action/Action/Fade.cpp

```diff
@@ -0,0 +1,75 @@
+#include "Main.h"
+#include "DxLib.h"
+#include "Fade.h"
+
+// ================================================================================
+// �t�F�[�h�C���A�t�F�[�h�A�E�g
+// ===============================================================================
+Fade::Fade()
+{
+	m_alpha = 0;
+	m_isFading = false;
+	m_fadeType = -1;
+}
+
+Fade::~Fade()
+{
+}
+
+void Fade::Start(enum FadeType fadeType_)
+{
+	switch (fadeType_)
+	{
+	case fadeIn:
+		m_alpha = 255;
+		m_isFading = true;
+		m_fadeType = 0;
```

Comment:
フェード種別を0/1で表現するのをやめてenumに変えたのですから、
```suggestion
		m_fadeType = fadeType;
```
では？
また、そうなると `m_fadeOut` もint型ではなくなりますね。

この部分は `case fadeOut` のときも同様なので、関数の最後で纏めてしまえば良さそうですね。
`m_isFading = true;` も共通なので纏めましょう。
DRYの原則です！！

Reply:
演算子を使って纏めました。